### ISSUE TITLE
No indication of how many items are contained in a menu

### ISSUE THREAD

**Issue by developpeurweb** [SAP/spartacus#13560]
**Describe the bug**
- Screen Reader vocalization of how many items are contained in the menus is flawed, or there is no indication of how many items are contained in the menu or at least a group in the menu is heard.

**Tell us the version of Spartacus**
- Library version: 4.0.

**To Reproduce**
1. Open any Screen Reader (VO, NVDA, JAWS)
2. Tab to Navigation
3. Open menu using keyboard only
4. Hear error.

**Expected behavior**
- SR vocalizes how many elements the dropdown group has. We have already achieved something similar with lists for search box suggestions, see screenshot below.

**Screenshots**
Currently no indication of how many items is heard.

![Screen Shot 2021-08-26 at 5 35 15 PM](https://user-images.githubusercontent.com/15113219/131042123-2dc665b9-cef0-4db9-bbff-2cf433a26352.png)

**Example of how we achieved this for search box suggestions**

![search available lists](https://user-images.githubusercontent.com/15113219/131042194-cd36b705-0398-4339-aa03-b8ad3c2c165a.jpg)

![items in list](https://user-images.githubusercontent.com/15113219/131042221-c1c689dc-3c8c-452f-aa7e-ed192d96d31b.jpg)

**Comment by developpeurweb** [comment]
It seems this is no longer an issue after navigation refactoring done in #13561 and #13511

Works well for VO/NVDA/JAWS

![2022-01-25_16-11-48](https://user-images.githubusercontent.com/15113219/151070249-b26e39ee-a99a-4162-99a9-5372ed4a8b14.jpg)
<img width="960" alt="2022-01-25_16-54-06" src="https://user-images.githubusercontent.com/15113219/151070253-8a749469-9018-472c-b33e-e9b46212db39.png">

JAWS doesn't have speech viewer, but I can confirm that I hear it correctly.

**Comment by KateChuen** [comment]
Tested on develop branch of spartacus using yarn
More extensive testing (on other screenreaders and devices) will be done with RC.

<img width="1128" alt="Screen Shot 2022-03-28 at 1 54 09 PM" src="https://user-images.githubusercontent.com/45409515/160458278-cff51327-6954-43e7-aa95-afc382ec53f7.png">

<img width="1167" alt="Screen Shot 2022-03-28 at 1 54 42 PM" src="https://user-images.githubusercontent.com/45409515/160458273-d5383965-2180-4203-a484-938abcf1f975.png">

<img width="1184" alt="Screen Shot 2022-03-28 at 1 55 09 PM" src="https://user-images.githubusercontent.com/45409515/160458263-85736c0c-59b4-4946-bff6-f876f345f323.png">
